modules/smtp/site.js

```diff
@@ -378,7 +378,7 @@ var copy_text_to_clipboard = function(e) {
 }
 
 var is_valid_recipient = function(recipient) {
-    var valid_regex = /^[\w ]*[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+    var valid_regex = /^[\p{L}' ]*[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/u;
```

Comment:
Why can't an email address start with a number? I am not sure what the unicode letter requirement in the beginning is used for? Maybe we need a more standard email checking regex?

Reply:
The name can't start with a number as well?

Reply:
I'm not aware of any valid person's name starting with a number, but if it's possible for such a name to be accepted, let's consider that possibility. By the way, the use of the Unicode character allows for the inclusion of letters from various languages worldwide, such as in the name 'André'.

Reply:
Even if it doesn't start with a letter itself, a name like Peter Parker 2nd <peter@parker.org> will not pass your new regex. I agree using unicode letters is a good thing but we should expand the range of matches in my opinion.